client/locations-list-map.js

```diff
@@ -66,9 +65,18 @@ let gLastLocationRendered = -1;
 
 const searchParams = new FtmUrl(window.location.href).searchparams;
 
+// Converts a string from 'a,b,c' to 'a, b, c'
+function addSpaceAfterComma(str) {
+  if (str) {
+    return str.split(',').join(', ');
```

Comment:
Could use a regex in the call to split, so you don't keep adding spaces if the source string already has them.

Reply:
Not as worried for now since it's a machine generated enum. Also it's html thus extra spaces collapse.